content/item-hash/index.md

```diff
@@ -0,0 +1,213 @@
+---
+rfc:
+start_date: 2018-08-03
+pr:
+status: draft
+---
+
+# Item hash with redaction
+
+## Summary
+
+This RFC proposes an algorithm to hash items such that data can be redacted
+without breaking the integrity of the register.
+
+**This RFC is not backwards compatible**
+
```

Comment:
Are there any security considerations for this RFC?

Reply:
What kind of security considerations are you thinking on?

Reply:
I don't have any specific concerns, it just seems like a section we should include in any RFC that touches on integrity guarantees or anything like that (I also noticed in the current spec, the security considerations section is blank). As a reader I don't know what I don't know, so even if there's nothing to say here it's good to have something that says so.